test/fixtures/movies.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
should have done `rails new <app_name> -BT` in the beginning. Now, it's generating many test files. If you want to test, use rspec.